docs/contextmanager.md

```diff
@@ -0,0 +1,67 @@
+The `KernelPool` context manager
+====================================================
+The KernelPool context manager provides a convenient way to load, and unload SPICE kernels, guaranteeing that the kernel database will still be cleared if an exception is raised.
+
```

Comment:
a warning here about how kclear is called before reloading the global kernels is needed here, 3-5 sentences explaining the most important caveats we discussed in the github issue discussion (e.g. any user defined variables in the kernel pool will be reset).

also a link to the naif's "required reading" on the kernel pool should be added.

Reply:
okay see that explained below, I'd just like a clear warning here using the warning rst syntax (https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html)

Reply:
I added the warning in the first section, and a note in the last one referring to the Kernel required reading for more information about the kernel pool, kernel's management, etc.